Engineering/QA/Boundary and Equivalence Partitioning Testing

```diff
@@ -0,0 +1,74 @@
+---
+Tags: Engineering/QA, Testing Type, QA Terms
+Author: Binh Le
+Date: 2022-10-15
+---
+# Boundary and Equivalence Partitioning Testing
```

Comment:
No need for h1 as it will be extracted from the file name when the note is published.

Reply:
Updated!